ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -311,6 +317,24 @@ public void OnBatchComplete()
                     }
                 }
             }
+
+            public void IncrementPendingJSCalls()
```

Comment:
>IncrementPendingJSCalls [](start = 24, length = 23)

In almost all cases, we haven't used acronyms for JS (I think originally due to some StyleCop rule we had been using that didn't allow acronyms, we no longer do this). But we should be consistent nonetheless. Since these are called `BridgeIdle` and `BridgeBusy`, I'd be fine with calling this `IncrementPendingBridgeCalls`, it's internal code anyway. No one is ever really expected to implement `IReactCallback`.

Reply:
I was just following upstream interface names to make tracking future changes easier, happy to change it.

Reply:
There are a few cases where we diverge from upstream to make things feel more "C#-y"

Reply:
#fixed